staging/src/k8s.io/apiserver/pkg/apis/audit/v1alpha1/types.go

```diff
@@ -216,10 +216,23 @@ type GroupResources struct {
 	// The empty string represents the core API group.
 	// +optional
 	Group string `json:"group,omitempty" protobuf:"bytes,1,opt,name=group"`
+
+	// NOTE: Servers that support v1beta1 also support subresource matching, but keep that
+	// undocumented since clients should explicitly switch to the new API group to get the
+	// new functionality.
+
 	// Resources is a list of resources within the API group.
 	// Any empty list implies every resource kind in the API group.
 	// +optional
 	Resources []string `json:"resources,omitempty" protobuf:"bytes,2,rep,name=resources"`
+	// The ResourceNames field was added for parody with the v1beta1 API group and is unsupported
```

Comment:
This is odd. Why do we have this field? In an alpha API group we can of course easily add anything. But what does "unsupported" mean?

The official way to add a field in a beta is to either
a) add it to the alpha too (what we do here)
b) store it in annotations for alpha during conversion.

With a) we completely support the new field also in the alpha.

Reply:
"unsupported" tries to convey that some API servers that support the v1alpha1 API group don't actually support this field (e.g. if you use a 1.7 server).

Might have overthought this one. Let me just add it to the v1alpha1 API group normally.

Reply:
I would still use b) in general. Here for alpha I don't care. But adding everything new to old apigroups feels wrong.

Reply:
+1 to b)

Reply:
doing a field -> annotation conversion means custom conversions... what is the benefit over adding the field? let's not spend more effort on v1alpha1 than required

Reply:
@liggitt that's what I meant with "for alpha I do not care". we will drop that anyway soon. Let's leave this additional field, but remove the strange comment about "not supported".

Reply:
Will remove the comment